spec/latest/json-ld-api/index.html

```diff
@@ -40,33 +40,48 @@
       githubAPI:              "https://api.github.com/repos/json-ld/json-ld.org",
       includePermalinks:      true,
       noRecTrack:             false,
-      testSuiteURIkey:       "http://json-ld.org/test-suite/", 
+      testSuiteURIkey:       "http://json-ld.org/test-suite/",
       postProcess:            [internalizeTermListReferences],
 
       // editors, add as many as you like
       // only "name" is required
       editors:  [
-          { name: "Markus Lanthaler", url: "http://www.markus-lanthaler.com/",
-            company: "Graz University of Technology", companyURL: "http://www.tugraz.at/" },
-          { name: "Gregg Kellogg", url: "http://greggkellogg.net/",
-            company: "Kellogg Associates", companyURL: "http://kellogg-assoc.com/" },
-          { name: "Manu Sporny", url: "http://manu.sporny.org/",
-            company: "Digital Bazaar", companyURL: "http://digitalbazaar.com/" }
+        { name:       "Gregg Kellogg",
+          url:        "http://greggkellogg.net/",
+          company:    "Spec-Ops",
+          companyURL: "https://spec-ops.io/",
+          w3cid:      "44770",
+          note:       "v1.0 and v1.1" }
       ],
 
       // authors, add as many as you like.
       // This is optional, uncomment if you have authors as well as editors.
       // only "name" is required. Same format as editors.
-
       authors:  [
-          { name: "Dave Longley", url: "http://digitalbazaar.com/",
-            company: "Digital Bazaar", companyURL: "http://digitalbazaar.com/"},
-          { name: "Gregg Kellogg", url: "http://greggkellogg.net/",
-            company: "Kellogg Associates", companyURL: "http://kellogg-assoc.com/" },
-          { name: "Markus Lanthaler", url: "http://www.markus-lanthaler.com/",
-            company: "Graz University of Technology", companyURL: "http://www.tugraz.at/" },
-          { name: "Manu Sporny", url: "http://digitalbazaar.com/",
-            company: "Digital Bazaar", companyURL: "http://digitalbazaar.com/" }
+        { name:       "Dave Longley",
+          url:        "http://digitalbazaar.com/",
+          company:    "Digital Bazaar",
+          companyURL: "http://digitalbazaar.com/",
+          note:       "v1.0" },
```

Comment:
Why do you note the versions for "authors"? The vast majority of the content didn't change between the versions